Data Exchange landing page and FAQ (#2246)
spryker/spryker-docs: docs/pbc/all/data-exchange/202311.0/data-exchange.md

<script type="text/javascript" src="https://viewer.diagrams.net/js/viewer-static.min.js"></script>

We recommend building custom integrations with the existing Alumio connectors in the following cases:
- There is no Spryker Integration App for the system you need. However, Alumio offers a connector for this system.
- You require a quick setup of the data exchange process, which should take up to a few days.
- You don't have the developer resources to do the coding job upon initial setup.
- You don't have the developer resources to do the coding job when you need to update mapping or configuration settings.
- You have a tech-savy person who can configure mapping of data for exchange between Spryker and a third-party system.
- For your project, it is important that the data exchange solution allows for the proper data orchestration like dependency management, error handling and logging, entity mapping, integration variable management, .CSV file validation, health monitoring, etc.

> For more details on how to manage integrations and exchange data in the Alumio platform, see QUESTION: WHAT LINK TO ALUMIO DOCUMENTATION SHOULD WE PROVIDE HERE?

Reviewer: ```suggestion
For more details on how to manage integrations and exchange data in the Alumio platform, see:
- [The Basic Parts Of A Connector Package In Alumio] (https://support.alumio.com/support/solutions/articles/80001031213-the-basic-parts-of-a-connector-package-in-alumio)
- [How To Use The Alumio Connector Packages](https://forum.alumio.com/t/how-to-use-the-alumio-connector-packages/219) 
- Alumio keeps here up-to-date the list of their standard Connector Packages [Latest iPaaS Connector Packages](https://forum.alumio.com/c/ipaas-connector-packages/27)
```

> ### Custom Integration Apps with custom connectors

Reviewer: ```suggestion
### Custom integrations with custom connectors
```

> 

Reviewer: ```suggestion
Before starting to implement any custom connector:
- Double check if the connector with the third-party system you want to integrate with is in the list of existing Alumio standard connectors: [Alumio iPaasS Connector Packages](https://forum.alumio.com/c/ipaas-connector-packages/27)  - this list is updated continuously

- If the system you wish to integrate with is not on the list, you may always submit a request to implement a standard connector, and see if it aligns with your timelines. Check here how to do it: [How do I request a new connector package?](https://forum.alumio.com/t/how-do-i-request-a-new-connector-package/148)

```

> If you have legacy systems or systems of old versions with which you need to exchange data and for which Alumio does not provide a connector, you can still build you integration apps. You can build a connector to these systems, as Spryker Middleware Powered by Alumio includes an SDK to build own connectors.

Reviewer: ```suggestion
If you have legacy systems or old version systems with which you need to exchange data, and for which Alumio does not provide a connector, you can still build your own connector to these systems, as Alumio platform includes an Alumio SDK to build your own connectors.
```

> To build an integration app for a legacy system, you need:

Reviewer: ```suggestion
To build a data integration for a legacy system, you need:
```

> 1. Provision the Spryker Middleware Powered by Alumio.

Reviewer: ```suggestion
1. Have a user account on Spryker Middleware powered by Alumio
```

> 2. Build a connector to the legacy system using the SDK from the Spryker Middleware Powered by Alumio. Building a connector with SDK usually takes up to a few weeks.

Reviewer: ```suggestion
2. Build a connector to the legacy system using the Alumio SDK. Building a connector with the Alumio SDK usually takes up to a few weeks.
```

3. Build the mapping and transformation of data. Initial configuration of data mapping and transformation usually takes up to a few days, changing them takes up to a few hours.

> With the Integration Apps with custom connectors, the data exchange process like this:

Reviewer: ```suggestion
With the data integration with custom connectors, the data exchange process looks like this:
```
